src/PharoThings-Hardware-Core.package/PotBoard.class/instance/extendConnector.with..st

```diff
@@ -0,0 +1,7 @@
+initialization
+extendConnector: aString with: pinsArray
+	| connector pinsArrayExtended |
+	pinsArrayExtended := self addPins: pinsArray named: aString.
+	connector := PotBoardConnector named: (aString) in: self with: pinsArrayExtended.
+	self replaceConnector: connector named: aString.
+	^connector
```

Comment:
@oliveiraallex please excuse me for being so critical but I am still not happy with this.
I would like to see following code:
```Smalltalk
Board>>extendConnector: aString with: pinsArray
 	| connector pinsArrayExtended |
 	connector := self connectorNamed: aString.
 	connector addPins: pinsArray.
 	^connector 

Connector>>addPins: anArray
    self pins: pins , anArray
```
It is simper. It does not require connectors replacement which is strange to see during initialization code. It does not expose pins to the board: concatenation of pins is incapsulated inside its owner - connector.  
Also notice that usually in Smalltalk the method with prefix #add is supposed to add something into receiver (modify receiver) instead of performing some summary operation and returning result.

Reply:
Don't worry @dionisiydk, thanks for the orientation. I was replacing the array inside the ordered collection "connectors" and it works, but it requires a bit more work, of course. 
Really the code above is simpler because recreate the "connectors" with #pins:  and not just replaces an array inside it.